scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -765,13 +765,13 @@ class Property:
       Convenience for spec.type.
 
     val_as_token:
```

Comment:
As you are breaking backwards compat of the signature anyway, you could also rename the member to "val_as_tokens" like you already did with "enum_indices".

Reply:
I changed it